docs/guides/qiskit-addons-sqd-get-started.ipynb

```diff
@@ -0,0 +1,1964 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting started with SQD\n",
+    "\n",
+    "In this page we will overview the `qiskit-addons-sqd` package and demonstrate a simple working example to get started. We will demonstrate this by using SQD to obtain an approximation of the ground state of the $N_2$ molecule at equilibrium.\n",
+    "\n",
+    "To begin, recall that the generic interacting-electron Hamiltonian has the form:\n",
+    "\n",
+    "$$ \\hat{H} = \\sum_{\\substack{pr\\\\ \\sigma}}h_{pr} \\hat{a}_{p\\sigma}^\\dagger \\hat{a}_{r\\sigma} + \\sum_{\\substack{prqs\\\\ \\sigma\\tau}} \\frac{1}{2}\\left(pr|qs\\right)\\hat{a}_{p\\sigma}^\\dagger \\hat{a}_{q\\tau}^\\dagger \\hat{a}_{s\\tau}\\hat{a}_{r\\sigma} $$\n",
+    "\n",
+    "where $\\hat{a}_{p\\sigma}^\\dagger$ and $\\hat{a}_{p\\sigma}$ are the fermionic creation and annihilation operators associated with the $p$-th orbital with spin $\\sigma$ and $h_{pr}$ and $\\left(pr|qs\\right)$ are the one and two-body electronic integrals."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<Admonition type = \"note\">\n",
+    "This example ingests pre-generated data in a file called `n2_fci.txt` which contains one and two-body electronic integrals. To run the code cells below on a local machine, simply copy and pasted the data into a file with the same name. This example also utilizes the `pyscf` package so be sure to install those in the environment used to execute it.\n",
+    "</Admonition>\n",
+    "\n",
+    "<details>\n",
+    "<summary>\n",
+    "Click here to examine the orbital information used to generate the ansatz\n",
+    "\n",
+    "</summary>\n",
```

Comment:
Can we add the code cell thingy where we have the name of the file in the code cell? Like we do for serverless pages?

Also i like the use of the twisty here! definitely makes it much more readable

Reply:
Do you mean putting the filename in the twisty codeblock or the first code cell where the file is ingested?